applications/system-service/fi.w1.wpa_supplicant1.xml

```diff
@@ -38,7 +38,7 @@
             <arg name="args" type="a{sv}" direction="in"/>
         </method>
         <method name="SignalPoll">
-            <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QVariantMap"/>
+            <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QVariant"/>
```

Comment:
I assume this change is not needed since a raw call is being made now.

Reply:
I didn't actually test without this change, but it's probably safer to leave it as is.